src/plugins/management/public/management_app.tsx

```diff
@@ -54,35 +54,38 @@ export class ManagementApp {
       title,
       mount: async ({}, params) => {
         let appUnmount: Unmount;
-        async function setBreadcrumbs(crumbs: ChromeBreadcrumb[]) {
-          const [coreStart] = await getStartServices();
-          coreStart.chrome.setBreadcrumbs([
-            {
-              text: i18n.translate('management.breadcrumb', {
-                defaultMessage: 'Management',
-              }),
-              href: '#/management',
-            },
-            ...crumbs,
-          ]);
-        }
-
-        ReactDOM.render(
-          <ManagementChrome
-            getSections={getSections}
-            selectedId={id}
-            legacySections={getLegacyManagementSections().items}
-            onMounted={async element => {
-              appUnmount = await mount({
-                basePath,
-                element,
-                setBreadcrumbs,
-              });
-            }}
-          />,
-          params.element
-        );
+        if (!this.enabledStatus) {
+          window.location.hash = '/management';
+        } else {
+          async function setBreadcrumbs(crumbs: ChromeBreadcrumb[]) {
+            const [coreStart] = await getStartServices();
+            coreStart.chrome.setBreadcrumbs([
+              {
+                text: i18n.translate('management.breadcrumb', {
+                  defaultMessage: 'Management',
+                }),
+                href: '#/management',
+              },
+              ...crumbs,
+            ]);
+          }
 
+          ReactDOM.render(
+            <ManagementChrome
+              getSections={getSections}
+              selectedId={id}
+              legacySections={getLegacyManagementSections().items}
+              onMounted={async element => {
+                appUnmount = await mount({
+                  basePath,
+                  element,
+                  setBreadcrumbs,
+                });
+              }}
+            />,
+            params.element
+          );
+        }
         return async () => {
```

Comment:
Seems `appUnmount()` and `ReactDOM.unmountComponentAtNode(params.element)` will be called even if React was not mounted. Maybe you could do an early return

```ts
if (!this.enabledStatus) {
  window.location.hash = '/management';
  // Early return
  return () => {};
}
```

and leave the rest of the function as before.